### PR TITLE
updating sticky nav bar for new navigation, removed temp nav from app.js

### LINKED ISSUE #7: Sticky Navigation Bar
Navigation bar that stays in the user’s view as the scroll through all pages of the website

### PR DESCRIPTION
**Pull request by alognz**
closes #7 

closed other pr bc i was having rebase issues that couldnt be fixed :D: but heres a gif of what the sticky nav bar looks like

stickynavbargid

### CODE REVIEW

File: src/components/stickyNavBar.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const StickyNavBar = () => {
+
+  return (
+    <>
+      <style>
```
Comment (boldenj2) [inline src/components/stickyNavBar.js]: For the sake of consistency and organization, can you put the styling in a `stickyNavBar.css` file

**Review by boldenj2 — CHANGES_REQUESTED** [review]
Other than the small nit comment, everything else looks good.

**Comment by alognz** [comment]
i cant make changes bc my push is being declined due to repository rule violations, ill close and make another branch tomorrow

**Comment by scuruchima1** [comment]
@alognz @boldenj2 the push might be prevented because of the unresolved conversation. other than that, Alondra you should be able to commit to the branch then commit after you resolve Bolden's comment. If not let me know.

**Review by scuruchima1 — APPROVED** [review]

**Review by boldenj2 — APPROVED** [review]

**Pull request merged**